metagraph/src/cli/stats.cpp

```diff
@@ -143,27 +166,35 @@ void print_stats(const Annotator &annotation) {
                                     / annotation.num_labels() << std::endl;
     std::cout << "representation: "
               << utils::split_string(annotation.file_extension(), ".").at(0) << std::endl;
+    using namespace annot::binmat;
 
-    if (const auto *rbmat = dynamic_cast<const annot::binmat::RainbowMatrix *>(&annotation.get_matrix())) {
+    if (const auto *rbmat = dynamic_cast<const RainbowMatrix *>(&annotation.get_matrix())) {
         std::cout << "================= RAINBOW MATRIX STATS =================" << std::endl;
         std::cout << "distinct rows: " << rbmat->num_distinct_rows() << std::endl;
-    }
 
-    if (const auto *brwt = dynamic_cast<const annot::binmat::BRWT *>(&annotation.get_matrix())) {
-        std::cout << "=================== Multi-BRWT STATS ===================" << std::endl;
-        std::cout << "num nodes: " << brwt->num_nodes() << std::endl;
-        std::cout << "avg arity: " << brwt->avg_arity() << std::endl;
-        std::cout << "shrinkage: " << brwt->shrinking_rate() << std::endl;
-        if (get_verbose()) {
-            std::cout << "==================== Multi-BRWT TREE ===================" << std::endl;
-            brwt->print_tree_structure(std::cout);
-        }
+    } else if (const auto *brwt = dynamic_cast<const BRWT *>(&annotation.get_matrix())) {
+        print_brwt_stats(*brwt);
+
+    } else if (const auto *brwt_rd
+               = dynamic_cast<const RowDiff<BRWT> *>(&annotation.get_matrix())) {
+        std::cout << "underlying matrix: BRWT" << std::endl;
```

Comment:
Isn't that enough that we print `std::cout << "representation: ...", which defines the underlying matrix?